test/fixtures/compound17.css

```diff
@@ -0,0 +1,49 @@
+
+:root
+{
+	--color: #ffffff;
+}
+
+#foo1 ul > li {
+	color: var(--color);
+}
+
+#foo2 ul ~ li {
+	color: var(--color);
+}
+
+#foo3 ul + li {
+	color: var(--color);
+}
+
+#foo1.bar ul > li {
+	--color: #111111;
+}
+
+#foo1.bar ul div > li {
+	--color: #777777;
+}
+
+#foo1.bar ul > div li {
+	--color: #888888;
+}
```

Comment:
I'm going back and forth on whether this should match 🤔 


```
From:
#foo1 ul > li

Matches:
#foo1.bar ul > li

Should this match?:
#foo1.bar ul > div li
```

In any case, a comment about why or known-limitation